src/content/article/kodi-21-0-omega-release/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Kodi 21.0 "Omega" - Release
+date: 2024-04-06T16:13:41.657Z
+author: Team Kodi
+tags:
+  - Release
+featured_image:
+  title: Kodi 21 "Omega" Splash Screen
+  alt: Kodi 21 "Omega" Splash Screen - the Kodi logo sits at the centre of a black
+    screen, a stylised Greek letter "Omega" around it.
+  src: /images/blog/kodi_21.0_omega_splash.webp
+---
+**It's time!** We are very pleased to present the latest and greatest release of Kodi, v21.0 "Omega".
+
+With this release we have seen over 3,750 commits since v20.0 "Nexus" was available to the world on January 15th, 2023. As always, this has been a huge effort from many contributors, testers and supporters. Thanks go out to our team members, plus everyone in our community who sent in a pull request, tested and gave feedback, or provided support to users on the forum. We always appreciate your contribution to making Kodi better.
+
+Let's review some of the changes you can expect:
+
+### Major Features
+
+**FFmpeg 6**
+
+Kodi relies significantly on FFmpeg to do a lot of heavy lifting for us. Many developers have contributed to upgrading the project to make use the the newer FFmpeg releases over the past 15 months, initially starting with targeting FFmpeg 5, but later updating to FFmpeg 6. We are always greatly appreciative of the work of all of the Open Source software projects that we rely on on our journey to bring you Kodi.
```

Comment:
```suggestion
Kodi relies significantly on FFmpeg to do a lot of heavy lifting for us. Many developers have contributed to upgrading the project to make use of the newer FFmpeg releases over the past 15 months, initially starting with targeting FFmpeg 5, but later updating to FFmpeg 6. We are always greatly appreciative of the work of all of the Open Source software projects that we rely on on our journey to bring you Kodi.
```
Also, might want to do something to remove the two "on"s in a row, e.g. simplify to "that we rely on to bring you Kodi."